saleor/graphql/checkout/tests/mutations/test_checkout_add_promo_code.py

```diff
@@ -1181,3 +1181,29 @@ def test_checkout_add_voucher_code_invalidates_price(
     assert data["checkout"]["voucherCode"] == voucher.code
     assert data["checkout"]["subtotalPrice"]["gross"]["amount"] == expected_total
     assert data["checkout"]["totalPrice"]["gross"]["amount"] == expected_total
+
+
+def test_new_error_flow(
```

Comment:
We should avoid using `new_error_flow` in tests. After some time we could have `newset_error_flow`. 
This name appears in many files.